test/integration/app.js

```diff
@@ -1,29 +1,25 @@
-/* eslint-disable no-unused-expressions */
 const nock = require('nock');
 const cheerio = require('cheerio');
 const chai = require('chai');
 const chaiHttp = require('chai-http');
 const server = require('../../server');
-const getSampleResponse = require('../resources/getSampleResponse');
 const constants = require('../../app/lib/constants');
 const messages = require('../../app/lib/messages');
+const getSampleResponse = require('../resources/getSampleResponse');
+const iExpect = require('../lib/expectations');
 
 const expect = chai.expect;
 
 chai.use(chaiHttp);
 
-function checkHtmlResponse(err, res) {
-  expect(err).to.be.null;
-  expect(res).to.have.status(200);
-  expect(res).to.be.html;
-}
-
 describe('redirection', () => {
   it('default should be find help page', (done) => {
     chai.request(server)
       .get('/')
       .end((err, res) => {
-        checkHtmlResponse(err, res);
+        iExpect.checkHtmlResponse(err, res);
```

Comment:
😄  Nice use of 'i'

Would read well as iExpect.htmlResponse200() or iExpect.htmlResponse({ expectedStatus: 200 });